Library/Homebrew/formula.rb

```diff
@@ -1629,7 +1629,12 @@ def std_configure_args
   end
 
   # Standard parameters for cargo builds.
-  sig { params(root: T.any(String, Pathname), path: String).returns(T::Array[T.any(String, Pathname)]) }
+  sig {
+    params(
+      root: T.any(String, Pathname),
+      path: T.any(String, Pathname),
+    ).returns(T::Array[T.any(String, Pathname)])
```

Comment:
Let's make this return an array of `String`s all the time by calling `#to_s` below.

Reply:
Updated, used interpolation to do this since (which also preserves the flags when logging `pretty_args` below in `system`)